lib/__tests__/ignore.test.js

```diff
@@ -191,8 +191,11 @@ describe("extending config with ignoreFiles glob ignoring one by negation", () =
 });
 
 describe("specified `ignorePath` file ignoring one file", () => {
-  let results;
+  const noFilesErrorMessage = new Error(
+    "The specified `files` glob returns no results"
```

Comment:
Shall we adopt the ESLint approach of?:

```
No files matching the pattern "${pattern}" were found.
```

This feels a little more explicit as we're stating the glob pattern in the error message, rather than just referring to it.